packages/data/src/settings/use-settings.js

```diff
@@ -30,7 +30,7 @@ export const useSettings = ( group, settingsKeys = [] ) => {
 				isDirty: getIsDirty( group, settingsKeys ),
 			};
 		},
-		[ group, settingsKeys ]
+		[ group, ...settingsKeys ]
```

Comment:
should we sort the settings keys alphabetically to make the most of useSelect's memoization?

Reply:
Sorting them makes an assumption that their order won't be predetermined already. But I could use json stringify instead which I almost did. Maybe we should do that

Reply:
> should we sort the settings keys alphabetically to make the most of useSelect's memoization?

Actually on second thought this sounds ok I think. Yep I'll do that.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin/pull/6540/commits/8ddc6f78ea67d1e5a6f9e7b226457ad96757eeb0 thanks for this suggestion 🙇